src/land/imsfv3_scf2ioda.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python3
+#
+# (C) Copyright 2021 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+
+import sys
+import argparse
+import netCDF4 as nc
+import numpy as np
+import re
+from datetime import datetime, timedelta
+import os
+from pathlib import Path
+
+IODA_CONV_PATH = Path(__file__).parent/"@SCRIPT_LIB_PATH@"
+if not IODA_CONV_PATH.is_dir():
+    IODA_CONV_PATH = Path(__file__).parent/'..'/'lib-python'
+sys.path.append(str(IODA_CONV_PATH.resolve()))
+
+import ioda_conv_engines as iconv
+from collections import defaultdict, OrderedDict
+from orddicts import DefaultOrderedDict
+
+locationKeyList = [
+    ("latitude", "float"),
+    ("longitude", "float"),
+    ("height", "float"),
+    ("datetime", "string")
+]
+
+obsvars = {
+    'snow_cover_fraction_': 'snowCoverFraction',
```

Comment:
Is there a mistake here with the underscore character at the end of the name?

Reply:
Good Catch. Thank you @gthompsnJCSDA. Removed _ now.